infrastructure/policy/iam.tf

```diff
@@ -17,7 +17,7 @@ resource "aws_s3_bucket_policy" "S3" {
             "AWS": "*"
```

Comment:
i think that maybe we can canfigure the policy access for specific resources like cloudformation and lambda